identity-service/src/models/index.js

```diff
@@ -9,8 +9,16 @@ const globalConfig = require('../config')
 const basename = path.basename(__filename)
 const db = {}
 
-// TODO(roneilr): set up debug-level logging of queries
-const sequelize = new Sequelize(globalConfig.get('dbUrl'), { logging: false, operatorsAliases: false })
+const sequelize = new Sequelize(globalConfig.get('dbUrl'), {
+  logging: false,
```

Comment:
why false for logging? don't we enable this for cnode..?

Reply:
@hareeshnagaraj enabled